Use man pages generated by help2man in the host build.

diff -u -r  ../zile-2.4.11/doc/Makefile.am ./doc/Makefile.am
--- ../zile-2.4.11/doc/Makefile.am	2014-02-24 21:42:16.000000000 +0000
+++ ./doc/Makefile.am	2015-11-14 19:40:13.163834200 +0000
@@ -27,11 +27,8 @@
 EXTRA_DIST += doc/man-extras doc/zile.1.in
 man_MANS = doc/zile.1
 
-doc/zile.1: $(srcdir)/doc/zile.1.in doc/Makefile.am config.status
-	@test -d doc || mkdir doc
-	rm -f $@ $@.tmp
-	$(edit) $(abs_srcdir)/doc/zile.1.in >$@.tmp
-	mv $@.tmp $@
+doc/zile.1:
+	cp $(top_srcdir)/../host-build/$@ $@
 
 $(srcdir)/doc/zile.1.in: doc/man-extras build-aux/zile-help2man-wrapper configure.ac $(builddir)/src/zile$(EXEEXT)
 	@test -d doc || mkdir doc
